blackbird_python/blackbird/blackbirdListener.py

```diff
@@ -242,30 +242,21 @@ def exitModes(self, ctx:blackbirdParser.ModesContext):
         pass
 
 
-    # Enter a parse tree produced by blackbirdParser#FunctionLabel.
-    def enterFunctionLabel(self, ctx:blackbirdParser.FunctionLabelContext):
-        pass
-
-    # Exit a parse tree produced by blackbirdParser#FunctionLabel.
-    def exitFunctionLabel(self, ctx:blackbirdParser.FunctionLabelContext):
-        pass
-
-
-    # Enter a parse tree produced by blackbirdParser#MulLabel.
-    def enterMulLabel(self, ctx:blackbirdParser.MulLabelContext):
+    # Enter a parse tree produced by blackbirdParser#BracketsLabel.
+    def enterBracketsLabel(self, ctx:blackbirdParser.BracketsLabelContext):
         pass
 
-    # Exit a parse tree produced by blackbirdParser#MulLabel.
-    def exitMulLabel(self, ctx:blackbirdParser.MulLabelContext):
+    # Exit a parse tree produced by blackbirdParser#BracketsLabel.
+    def exitBracketsLabel(self, ctx:blackbirdParser.BracketsLabelContext):
```

Comment:
I think(?) these are autogenerated, but just curious: what changed?

Reply:
I think everything got shifted down in the autogenerated files to make room for the new functions!